www/index.html

```diff
@@ -128,7 +129,6 @@
   </div>
 
   <!-- duplicate the default cross-platform instructions -->
-  <div>
```

Comment:
The reason we need to remove it is to ensure the content in the instructions is centered. Compared to the default `platform-instructions-unix`, we have this extra div here. Removing it will help ensure the style is applied correctly.